packages/lumx-react/src/components/user-block/UserBlock.tsx

```diff
@@ -18,6 +24,8 @@ export type UserBlockSize = Extract<Size, 's' | 'm' | 'l'>;
 export interface UserBlockProps extends GenericProps {
     /** Props to pass to the avatar. */
     avatarProps?: AvatarProps;
+    /** Props to pass to the avatar. */
+    linkProps?: LinkPropsBis;
```

Comment:
Same as the thumbnail and the avatar (and existing components). I'd rather have the `linkAs` outside the `linkProps`